config.example.yaml

```diff
@@ -146,6 +146,8 @@ services:
     conf: 
       port: 7231
       spec: *spec
+      salt: secret
+      default_page_size: 1000
```

Comment:
I used 1000 here because of this limit - https://github.com/wikimedia/restbase/pull/245/files#diff-159808c267853355eee587915c06aae6L317

Right now default_page_size is been used by those end points which make request to cassandra backend, do you want me to use it for other ends points as well?